Sources/ArcGISToolkit/Extensions/Foundation/UserDefaults.swift

```diff
@@ -0,0 +1,22 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+
+extension UserDefaults {
+    /// The value of the `-testCase` launch argument.
+    var testCase: String? {
+        string(forKey: "testCase")
+    }
+}
```

Comment:
Consider using `CommandLine.arguments` to read the test case. My understanding is that this behavior of `UserDefaults` is intended to allow overriding defaults using command line arguments. Since `testCase` isn't ever stored as a user default, `CommandLine.arguments` would be a better fit. Also, this code should be conditionally compiled for `DEBUG` (again, assuming UI tests are run in debug mode, which I am almost certain that they are).